sections/predictive-search.liquid

```diff
@@ -1,71 +1,149 @@
-          >
-            {%- if product.featured_media != blank -%}
-              <img
-                class="predictive-search__image"
-                src="{{ product.featured_media | image_url: width: 150 }}"
-                alt="{{ product.featured_media.alt }}"
-                width="50"
-                height="{{ 50 | divided_by: product.featured_media.preview_image.aspect_ratio }}"
-              >
-            {%- endif -%}
-            <div class="predictive-search__item-content{% unless settings.predictive_search_show_vendor or settings.predictive_search_show_price %} predictive-search__item-content--centered{% endunless %}">
-              {%- if settings.predictive_search_show_vendor -%}
-                <span class="visually-hidden">{{ 'accessibility.vendor' | t }}</span>
-                <div class="predictive-search__item-vendor caption-with-letter-spacing">{{ product.vendor }}</div>
-              {%- endif -%}
-              <h3 class="predictive-search__item-heading h5">{{ product.title }}</h3>
-              {%- if settings.predictive_search_show_price -%}
-                {% render 'price', product: product, use_variant: true, show_badges: false %}
-              {%- endif -%}
+      {%- if first_column_results_size > 0 -%}
+        <div class="predictive-search__result-group">
+      {%- endif -%}
+      {%- if predictive_search.resources.queries.size > 0 or predictive_search.resources.collections.size > 0 -%}
+        <div>
+          <h2 id="predictive-search-queries" class="predictive-search__heading text-body caption-with-letter-spacing">
+            {{ 'templates.search.suggestions' | t }}
+            <svg aria-hidden="true" focusable="false" role="presentation" class="spinner" viewBox="0 0 66 66" xmlns="http://www.w3.org/2000/svg">
+              <circle class="path" fill="none" stroke-width="6" cx="33" cy="33" r="30"></circle>
+            </svg>
+          </h2>
+          <ul id="predictive-search-results-queries-list" class="predictive-search__results-list list-unstyled" role="listbox" aria-labelledby="predictive-search-queries">
+            {%- for query in predictive_search.resources.queries -%}
+              <li id="predictive-search-option-A{{ forloop.index }}" class="predictive-search__list-item">
+                <a href="{{ query.url }}" class="predictive-search__item link link--text" tabindex="-1" role="option" aria-selected="false">
+                  <div class="predictive-search__item-content predictive-search__item-content--centered">
+                    <h3 class="predictive-search__item-heading predictive-search__item-query-result h5">{{ query.styled_text }}</h3>
+                  </div>
+                </a>
+              </li>
+            {%- endfor -%}
+            {%- for collection in predictive_search.resources.collections -%}
+              <li id="predictive-search-option-A{{ forloop.index | plus: predictive_search.resources.queries.size }}" class="predictive-search__list-item">
```

Comment:
Related to my comment above about the duplicated IDs: you could avoid the `plus:` here by adding the resource name. I don't think you actually need `plus` as you are relying on that number in your JS.

```suggestion
              <li id="predictive-search-option-{{ forloop.index }}-collection-A" class="predictive-search__list-item">
```

Reply:
Same comment everywhere else `predictive-search-option-A{{ forloop.index` is used